turbo/execution/eth1/ethereum_execution.go

```diff
@@ -122,16 +122,23 @@ func NewEthereumExecutionModule(blockReader services.FullBlockReader, db kv.RwDB
 }
 
 func (e *EthereumExecutionModule) getHeader(ctx context.Context, tx kv.Tx, blockHash libcommon.Hash, blockNumber uint64) (*types.Header, error) {
-	td, err := rawdb.ReadTd(tx, blockHash, blockNumber)
-	if err != nil {
-		return nil, err
-	}
-	if td == nil {
-		return nil, nil
+	// the polygon interlal sync process uses this method to retrive headers
```

Comment:
I don't think we need to explain polygon specific usages here in the EthereumExecutionModule as that is a leaking chain specific ideas in what ought to be a chain agnostic module.

TD is now pruned and only kept for the last 150_000 blocks so I think we can just remove the ReadTd code from here entirely without giving any explanations.